test/python/compiler/test_assembler.py

```diff
@@ -735,54 +735,65 @@ def test_assemble_backend_rep_times_delays(self):
         """Check that rep_time and rep_delay are properly set from backend values."""
         # use first entry from allowed backend values
         rep_times = [2.0, 3.0, 4.0]  # sec
-        rep_delays = [2.5e-3, 3.5e-3, 4.5e-3]
+        rep_delay_range = [2.5e-3, 4.5e-3]
```

Comment:
Do these test that the output dictionary has the right units and format?

Reply:
No, added one in `test/python/providers/test_backendconfiguration.py`. See https://github.com/Qiskit/qiskit-terra/pull/4830/commits/e46d7a001b29cb978a7fe46db96c155c36d9e2ea.